.github/workflows/test.yml

```diff
@@ -80,11 +84,11 @@ jobs:
 
     - name: >-
         Initialize tox envs
-      run: >-
-        python3 -m tox --notest --skip-missing-interpreters false -vv
-      env: ${{ matrix.env }}
+      run: |
```

Comment:
`set -ex` not echo if you want to make the commands visible.

Reply:
I changed it as suggested.